stable/datadog/templates/confd-configmap.yaml

```diff
@@ -23,3 +23,7 @@ override duplicates
 {{ tpl (toYaml .Values.datadog.confd) . | indent 2 }}
 {{- end -}}
 {{- end -}}
+  {{- if .Values.jankSecretsCopier }}
```

Comment:
you can technically left justify the `if` and `end` blocks

(the `- if` means delete all whitespace before it)